### PR TITLE
Updating members list

### PR DESCRIPTION
**Pull request by jodiedoubleday**
Technical change to JOB roles, removal of non-employed team members, adding PR template.

#### Author
- [x] I have checked for grammatical errors
- [x] I have checked for spelling errors
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 1
- [x] All information (technical or otherwise) is correct to my knowledge

#### Editor 2
- [x] All information (technical or otherwise) is correct to my knowledge

### CODE REVIEW

**Comment by DuncanFenning** [comment]
looks good 👍

File: _data/members.yml [hunk 1]
```diff
@@ -12,7 +16,7 @@ alanna_mooney:
   twitter: alannamooney
 alix_george:
   name: Alix George
-  role: Product Owner
+  role: Product Lead
```
Comment (dancork) [inline _data/members.yml]: Is Alix not Product Owner Lead?

**Comment by dancork** [comment]
Looks good, just 2 minor comments :+1:

File: _data/members.yml (hunk 1 above)
Comment (jodiedoubleday) [inline _data/members.yml]: nope

**Pull request merged**